### PR TITLE
Update testdriver to support computedrole/computedlabel

### PR DESCRIPTION
**Pull request by cookiecrook**
Closes https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/5

Includes basic explicit and implcit role/label tests each for accname (in /accname) and aria role computation (in /aria/role)

### CODE REVIEW

File: resources/testdriver.js [hunk 1]
```diff
@@ -220,6 +220,40 @@
...
+        /**
+         * Get Computed Label for an element.
+         *
+         * This matches the behaviour of the
+         * `Get Computed Label
+         * <https://w3c.github.io/webdriver/#dfn-get-computed-label>`_
+         * WebDriver command.
+         *
+         * @param {Element} element 
+         * @returns {Promise} fulfilled after the computed label is returned, or
```
Comment (cookiecrook) [inline resources/testdriver.js:232]:
```suggestion
         * @returns {Promise} fulfilled after the computed label is returned, or
```

Comment (cookiecrook) [inline resources/testdriver.js:232]:
```suggestion
         * @returns {Promise} fulfilled after the computed label is returned, or
```

File: resources/testdriver.js [hunk 2]
```diff
@@ -220,6 +220,40 @@
...
+        /**
+         * Get Computed Role for an element.
+         *
+         * This matches the behaviour of the
+         * `Get Computed Label
+         * <https://w3c.github.io/webdriver/#dfn-get-computed-role>`_
+         * WebDriver command.
+         *
+         * @param {Element} element 
+         * @returns {Promise} fulfilled after the computed role is returned, or
```
Comment (cookiecrook) [inline resources/testdriver.js:249]:
```suggestion
         * @returns {Promise} fulfilled after the computed role is returned, or
```

File: resources/testdriver.js [hunk 3]
```diff
@@ -220,6 +220,40 @@
...
+
+        /**
+         * Get Computed Role for an element.
+         *
+         * This matches the behaviour of the
+         * `Get Computed Label
+         * <https://w3c.github.io/webdriver/#dfn-get-computed-role>`_
+         * WebDriver command.
+         *
+         * @param {Element} element
```
Comment (cookiecrook) [inline resources/testdriver.js:248]:
```suggestion
         * @param {Element} element
```

File: resources/testdriver.js [hunk 4]
```diff
@@ -220,6 +220,40 @@
...
 
+        /**
+         * Get Computed Label for an element.
+         *
+         * This matches the behaviour of the
+         * `Get Computed Label
+         * <https://w3c.github.io/webdriver/#dfn-get-computed-label>`_
+         * WebDriver command.
+         *
+         * @param {Element} element
```
Comment (cookiecrook) [inline resources/testdriver.js:231]:
```suggestion
         * @param {Element} element
```

**Comment by cookiecrook** [comment]
If I'm reading the log correctly, the Azure Pipeline check may be spurious. `/infrastructure/testdriver/actions/actionsWithKeyPressed.html` passes for me with or without my change.

**Comment by cookiecrook** [comment]
The test failures are likely resolved by #38789

**Comment by cookiecrook** [comment]
Note: I need to move `/aria/role/basic.html` into `/wai-aria/role/basic.html` but otherwise this is ready for review.

**Review by gsnedders — APPROVED** [review]

**Pull request merged**